app/views/blogs/new.html.haml

```diff
@@ -0,0 +1,25 @@
+- if !user_signed_in?
+  %p
```

Comment:
Transfer this code in the application.html.haml file since this is a header code.